content/en/docs/tasks/administer-cluster/enabling-service-topology.md

```diff
@@ -8,8 +8,11 @@ content_type: task
 ---
 
 <!-- overview -->
-This page provides an overview of enabling Service Topology in Kubernetes.
+{{< feature-state for_k8s_version="v1.21" state="deprecated" >}}
 
+ServiceTopology is deprecated as of Kubernetes v1.21, and will be removed in
```

Comment:
Same as above - can we be specific about which API fields?

Reply:
Updated, thanks!